extension/src/test/resources/copy-all-sources.xlf

```diff
@@ -8,6 +8,7 @@
         <trans-unit id="Table 2328808854 - Field 1296262999 - Property 2879900210" size-unit="char" translate="yes" xml:space="preserve">
           <source>No Target</source>
           <note from="Developer" annotates="general" priority="2"></note>
+          <note from="NAB AL Tool Refresh Xlf" annotates="general" priority="3">New translation.</note>
```

Comment:
Should this really be here?

Reply:
Nah, I had some idea before but I'm pretty happy with the tests now. Removed :)